src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -109,10 +137,56 @@ void EnMag_Init(Actor* thisx, PlayState* play) {
 void EnMag_Destroy(Actor* thisx, PlayState* play) {
 }
 
+#if OOT_VERSION < OOT_GC_US
+void EnMag_CheckSramResetCode(PlayState* play, EnMag* this) {
+    static s32 sramResetCode[] = {
+        BTN_DUP, BTN_DDOWN,  BTN_DLEFT, BTN_DRIGHT, BTN_START, BTN_B, BTN_CDOWN,
+        BTN_L,   BTN_CRIGHT, BTN_CLEFT, BTN_A,      BTN_CUP,   BTN_R, BTN_Z,
+    };
+    s32 var_v1;
+
+    var_v1 = play->state.input[2].cur.button & 0xFF3F;
```

Comment:
This is holding the current controller 2 buttons pressed, minus the 0x40 and 0x80 "button" signals.
I asked tharo what these mean, and 0x80 is the controller reset signal, when you press L+R+Start. 0x40 is not known to have any significant meaning.
MM apparently has `#define BTN_RESET 0x0080`, but dunno if we want that.

I also wouldnt mind this happening on the same line
```suggestion
    s32 cont2CurBtns = play->state.input[2].cur.button & ~(0x40 | 0x80);
```

Reply:
I don't really follow the logic of this code in detail, but I think the "right" thing to do is probably to or all the button values together

Reply:
Yeah or-ing all buttons makes sense, changed to that (also this is controller 3 right?)

Reply:
It is controller 3 yes

Reply:
oh whoops it is 3, yeah.
I think its fine to name it `cont3CurBtns` or `cont3CurButtons` now, but w/e can also be done with further documentation

Reply:
Oh I totally missed the name change, sorry. It's a bit weird because `var_v1` is being modified in this function so I'm not sure if the name still applies after that. Like, maybe it would be more accurate to just call this "buttons". So yeah it might be best to name it at the same time as the rest (unk_E316/unk_E31C/unk_E320)